packages/location-field/src/options.js

```diff
@@ -31,6 +57,8 @@ export function Option({ disabled, icon, isActive, onClick, title }) {
 }
 
 Option.propTypes = {
+  classes: PropTypes.string,
```

Comment:
Add JSDoc to this prop.

Reply:
Added in e0fc8a811358ed8a55a587f948db1614da68dbd4